### PR TITLE
PSP-8412 | Updated lease header files and initial work on other file headers

### PR DESCRIPTION
**Pull request by FuriousLlama**

### CODE REVIEW

**Comment by devinleighsmith** [comment]
If it's all the same to you, I think I'll wait to review this one until the other header pr is merged.

**Comment by FuriousLlama** [comment]
> If it's all the same to you, I think I'll wait to review this one until the other header pr is merged.

This PR needs to be reviewed first actually.

**Review by devinleighsmith — CHANGES_REQUESTED** [review]
Same as other PR, just needs the project header changes.

File: source/frontend/src/index.tsx [hunk 1]
```diff
@@ -46,6 +46,8 @@ const Index = () => {
   );
 };
 
+console.log(bcTokens);
```
Comment (devinleighsmith) [inline source/frontend/src/index.tsx:49]: console.log

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**